Formula/iso-codes.rb

```diff
@@ -14,9 +15,17 @@ class IsoCodes < Formula
   end
 
   depends_on "gettext" => :build
+  depends_on :python3
+  depends_on "pkg-config" => :run
 
   def install
     system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "check"
     system "make", "install"
   end
+
+  test do
+    assert_match /iso_639-2 iso_639-3 iso_639-5 iso_3166-1/, shell_output("#{Formula["pkg-config"].opt_bin}/pkg-config --variable=domains iso-codes")
```

Comment:
Yes, that's clearly proportional to my suggestion that we add a single extra line used in 140+ other places.